hazelcast/src/main/java/com/hazelcast/internal/util/phonehome/MapInfoCollector.java

```diff
@@ -103,4 +105,18 @@ private long countMapWithNativeInMemoryFormat() {
         return mapConfigs.stream()
                 .filter(mapConfig -> mapConfig != null && mapConfig.getInMemoryFormat() == InMemoryFormat.NATIVE).count();
     }
+
+    private long MapPutLatency(Node node) {
+        return (long) mapConfigs.stream().map(mapConfig -> node.hazelcastInstance.getMap(mapConfig.getName()))
+                .filter(map -> map.getLocalMapStats().getPutOperationCount() != 0L)
```

Comment:
before this filter, you could map to `.map(IMap::getLocalMapStats)` because you don't need the map itself in the rest of the operations